InnerEye/ML/SSL/lightning_containers/ssl_container.py

```diff
@@ -223,25 +225,37 @@ def _get_transforms(self, augmentation_config: Optional[CfgNode],
         examples.
         :param dataset_name: name of the dataset, value has to be in SSLDatasetName, determines which transformation
         pipeline to return.
-        :param is_ssl_encoder_module: if True the transformation pipeline will yield two version of the image it is
-        applied on. If False, return only one transformation.
+        :param is_ssl_encoder_module: if True the transformation pipeline will yield two versions of the image it is
+        applied on and it applies the same transformations for validation. If False, return only one transformation.
```

Comment:
If think the "applies the same transformations for validation" depends on what validation transform the user defines. If the user chooses to apply the same transform at training and validation time, the pipeline would return two versions of the image. Maybe we can say something like "the pipeline will apply return two versions of the input image by applying the train (resp. validation) transformations twice. Note that if your transformation does not contain any randomness, the pipeline will return two identical copies."

Reply:
use_training_augmentations_for_validation=is_ssl_encoder , this is why I said it will apply the "same". I hear you on the randomness part. Changed as below, let me know if it's clear enough
"if True the transformation pipeline will yield two versions of the image it is applied on and it applies the same transformations for training and validation. Note that if your transformation does not contain any randomness, the pipeline will return two identical copies. If False, it will return only one transformation."

Reply:
Mmm i actually don't understand what you mean by "it applies the same transformations for training and validation"